game/ato/flightstate/navigating.py

```diff
@@ -29,11 +29,8 @@ def on_game_tick(
             events.update_flight_position(self.flight, self.estimate_position())
 
     def progress(self) -> float:
-        # if next waypoint is very close, assume we reach it immediately to avoid divide
-        # by zero error
-        if self.total_time_to_next_waypoint.total_seconds() < 1:
-            return 1.0
-
+        if self.total_time_to_next_waypoint.total_seconds() == 0.0:
+            return 99.9
```

Comment:
We should figure out whether Pretense depends on this, otherwise we should probably revert this part

Reply:
Reverted